main.py

```diff
@@ -1,10 +1,41 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as json_file:
+        players_data = json.load(json_file)
+
+    for player_name, player_data in players_data.items():
+        race, is_race_created = Race.objects.get_or_create(
```

Comment:
There is no need to create `is_race_created` variable in case it is not used. You can skip it here and in the other places as well
```suggestion
        race, _ = Race.objects.get_or_create(
```